gammapy/modeling/fit.py

```diff
@@ -326,7 +326,8 @@ def stat_profile(
         Returns
         -------
         results : dict
-            Dictionary with keys "values" and "stat".
+            Dictionary with keys "values", "stat" and "results". The latter contains an
```

Comment:
Hi @luca-giunti ! I find it a bit uncomfortable to have a `results` key inside a `results` dict. Probably it is better to remake it and explain what it means?

Reply:
Ah right. I could rename the dict key to `"fit_results"`. Would it be Ok then?